libraries/AP_HAL_ChibiOS/Storage.h

```diff
@@ -48,7 +48,13 @@ class ChibiOS::Storage : public AP_HAL::Storage {
     bool healthy(void) override;
 
 private:
-    volatile bool _initialised;
+    enum StorageBackend {
+        None,
+        FRAM,
+        Flash,
+        SDCard,
+    };
+    volatile enum StorageBackend _initialisedType = StorageBackend::None;
```

Comment:
don't think you need the enum here

Reply:
Don't need the `enum` now that `StorageBackend` is an `enum class`. Fixed.